src/server/schema.ts

```diff
@@ -0,0 +1,24 @@
+export const schema = `
+  type Source {
+    identifier: ID!
+    name: String!
+  }
+
+  type Term {
+    uri: ID!
+    prefLabel: [String]!
+    altLabel: [String]!
+    hiddenLabel: [String]!
+    scopeNote: [String]!
```

Comment:
The [current GraphQL API](http://demo.netwerkdigitaalerfgoed.nl:8080/static/graphiql/index.html?query=query%20%7B%0A%20%20__type(name%3A%20%22Term%22)%20%7B%0A%20%20%20%20fields%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%7D%0A%20%20%7D%20%20%20%20%20%0A%7D) also has the fields `definition`, `broader`, `narrower`, `related`.

Do we need these fields if we want to replace the currentAPI?

Reply:
Good one. We can do without them for now (the absence of these fields doesn't change the technical working of the prototype), but I'll add these soon.